apps/condo/webhooks/webhooks.test.js

```diff
@@ -0,0 +1,26 @@
+const { makeLoggedInAdminClient, makeClient } = require('@condo/keystone/test.utils')
+const { makeClientWithSupportUser } = require('@condo/domains/user/utils/testSchema')
+const { makeClientWithNewRegisteredAndLoggedInUser } = require('@condo/domains/user/utils/testSchema')
+const { WebhookTests } = require('@condo/webhooks/schema/models/Webhook.test')
+const { WebhookSubscriptionTests } = require('@condo/webhooks/schema/models/WebhookSubscription.test')
+
+async function initializeActors () {
```

Comment:
Storing it here is `OK`?

Reply:
Moved tests to `common` domain

Reply:
validator is still under discussion

Reply:
changed logic of initialising validator in: https://github.com/open-condo-software/condo/pull/2497/commits/86e9c28e85f95fab65325596b6d4e74903d0d7c9